src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -167,7 +171,7 @@ angular.module 'mnoEnterpriseAngular'
 
         MnoConfirm.showModal(modalOptions).then(
           (response) ->
-            vm.reviews.list.splice(key, 1)
+            vm.reviews.pageChangedCb(vm.app.id, vm.reviews.nbItems, vm.reviews.page)
```

Comment:
The pagination does not work very well with unshift and splice, it produces a bug.

-When deleting the last comment or last question in any page > 1, it will show a message "no reviews" or "no questions", when there are still in other pages.

That is the best reasonable solution I found.

Thoughts?